server/instances.go

```diff
@@ -417,7 +418,7 @@ func (p *Plugin) httpAutocompleteInstalledInstanceWithAlias(w http.ResponseWrite
 	}
 
 	for _, instanceID := range info.Instances.IDs() {
-		item := instances.getAlias(instanceID)
+		item := fmt.Sprintf("%s(%s)", instances.getAlias(instanceID), instanceID)
```

Comment:
Shouldn't there be a space between the alias and url?

Reply:
I think we can put the url as a help text

Reply:
![Screenshot from 2024-07-30 21-40-48](https://github.com/user-attachments/assets/5c4bbd5c-9adb-4829-ba8a-fa8ba63ef090)

Reply:
Even better. :+1: